### PR TITLE
Bumps patternfly versions to 2020.07

### PR DESCRIPTION
**Pull request by benthomasson**
This PR attempts to meet the CRC platform requirement to upgrade patternfly to 2020.07. 
It is going to fail for a while, while we track down all the changes.

### CODE REVIEW

**Comment by jeanetteobr** [comment]
Linking the document of proposed changes to components for the upgrade here: https://docs.google.com/document/d/1y2_osLrXUAggT-zrBMOYd0M8Qcgr7HL1OZpaTw8XcG4/edit?usp=sharing

**Comment by jeanetteobr** [comment]
Just an FYI: I bumped up our redhat-cloud-services deps (`"@redhat-cloud-services/frontend-components": "2.0.6", "@redhat-cloud-services/frontend-components-utilities": "2.0.4"`) to the latest version because the PageHeaderTitle component refused tp render properly. I blew away my node_modules and did an npm install afterwards. My code is failing to compile because it can't resolve DataToolBar and ChipGroup imports. Per Ryan Long, the team should be releasing a fix either today or tomorrow.

**Comment by jeanetteobr** [comment]
One of our unit tests is failing due to a missing import from the CRC package we're using. See below: 
    Cannot find module '@patternfly/react-core/dist/js/layouts/Toolbar/Toolbar.js' from 'index.js'

    However, Jest was able to find:
    	'./AutomationCalculator.js'
    	'./AutomationCalculator.test.js'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'jsx', 'ts', 'tsx', 'node'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:230:17)
      at node_modules/@redhat-cloud-services/frontend-components/index.js:1:2121
      at Object.<anonymous> (node_modules/@redhat-cloud-services/frontend-components/index.js:1:10279)`

Ryan Long is looking into this issue. Once a fix has been pushed we should be good to move along with deployment to CI for testing!

**Comment by kialam** [comment]
Noticed a bug on the org stats page where you can't toggle the first org item on the grouped bar chart:

org_toggle_pf_upgrade

**Comment by jeanetteobr** [comment]
@kialam Thanks! I'll look into it.

**Comment by kialam** [comment]
> @kialam Thanks! I'll look into it.

False alarm :)

Looks like it's a problem with the data generator creating an org with a `null` id. cc @benthomasson 

Screen Shot 2020-06-15 at 11 39 56 AM

**Comment by kialam** [comment]
@benthomasson @jeanetteobr this PR looks good to me; is there a way we can deploy this branch to CI and have QE test?

File: .cache_14vhhmn [hunk 1]
```diff
@@ -0,0 +1 @@
...ndColumn":59,"severity":1,"nodeType":null},{"ruleId":null,"message":"162","line":1,"column":1,"endLine":1,"endColumn":31,"severity":1,"nodeType":null},{"ruleId":null,"message":"161","line":1,"column":1,"endLine":1,"endColumn":82,"severity":1,"nodeType":null},{"ruleId":null,"message":"161","line":1,"column":1,"endLine":1,"endColumn":55,"severity":1,"nodeType":null},{"ruleId":null,"message":"161","line":1,"column":1,"endLine":1,"endColumn":82,"severity":1,"nodeType":null},{"ruleId":null,"message":"161","line":2,"column":1,"endLine":2,"endColumn":76,"severity":1,"nodeType":null},{"ruleId":null,"message":"161","line":1,"column":1,"endLine":1,"endColumn":82,"severity":1,"nodeType":null},{"ruleId":null,"message":"163","line":1,"column":1,"endLine":1,"endColumn":32,"severity":1,"nodeType":null},"'/*eslint*/' has no effect because you have 'noInlineConfig' setting in your config (--config » plugin:pf-codemods/recommended).","'/*eslint-disable*/' has no effect because you have 'noInlineConfig' setting in your config (--config » plugin:pf-codemods/recommended).","'/*global*/' has no effect because you have 'noInlineConfig' setting in your config (--config » plugin:pf-codemods/recommended)."]
```
Comment (benthomasson) [inline .cache_14vhhmn:1]: Please remove this cache file.

Comment (jeanetteobr) [reply .cache_14vhhmn:1]: On it!

**Pull request merged**